projects/clr/hipamd/src/hip_platform.hpp

```diff
@@ -130,6 +130,15 @@ class PlatformState {
     return false;
   }
 
+  inline bool GetLibraryFunction(const hipKernel_t f, hipLibrary_t* lib) {
```

Comment:
This function is called `GetLibraryFunction` but we output the function's library.
Can we rename it?